packages/api/test/__fixtures__/sdk/petstore/index.ts

```diff
@@ -1,5 +1,5 @@
 import type * as types from './types';
-import type { ConfigOptions, FetchResponse } from 'api/dist/core';
+import type { ConfigOptions, FetchResponse } from 'api/dist/core'
```

Comment:
It's kinda funky that these type imports are no longer getting codegenerated with a `;` but it seems like they do if there's three types being imported here. This might be resolved in the latest `ts-morph` releases, which we're a couple majors behind.

But since this code still works I'm not going to fret over it now.